docs/src/appendix/forge-library/declare.md

```diff
@@ -1,8 +1,8 @@
 # `declare`
 
-> `fn declare(contract: felt252) -> ClassHash`
+> `fn declare(contract: felt252) -> ContractClass`
 
-Declares a contract and returns its class hash.
+Declares a contract and returns `ContractClass`.
```

Comment:
Would be nice to describe what `ContractClass` is here as well - a struct with xyz fields and that you can call precalculate_address and deploy on its instance.